NYPLAudiobookToolkit/UI/AudiobookTrackTableViewCell.swift

```diff
@@ -15,13 +15,7 @@ class AudiobookTrackTableViewCell: UITableViewCell {
         let title = spineElement.chapter.title
         let detailLabel: String
         let labelAlpha: CGFloat
-        if progress == 0 {
```

Comment:
This block of code is basically the same as the last block, except the value of `labelAlpha`.

Reply:
would it make sense to remove the duplication using a private function for the same code?